.github/workflows/main.yml

```diff
@@ -242,11 +266,28 @@ jobs:
           pip3 install .[test]
       - run: pytest -s tests
 
+  semantic-release:
+    name: 🤖 Semantic release
+    needs: [rust-build-check, go-build-check, meson-build, js-build-check, linux-crypto-check, macosx-crypto-check]
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_name == 'master' && github.event_name == 'push' }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+      - run: yarn
+      - name: Tag release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          npx semantic-release
+
   python-build-check:
     name: 🐍 Python build & checks
-    needs: [reuse, c-lint, lua-lint]
+    needs: [semantic-version]
```

Comment:
Sure that this is the correct build deps? I just see a semantic-release job name